ruff.toml

```diff
@@ -0,0 +1,131 @@
+    "N805",  # no-method-argument
+    "N805",  # no-self-argument
+    "N815",  # invalid-name
+    "PGH001",  # eval-used
+    "PIE790",  # unnecessary-pass
+    "PLE0101",  # return-in-init
+    "PLE0116",  # continue-in-finally
+    "PLE0241",  # duplicate-bases
+    "PLE0302",  # unexpected-special-method-signature
+    "PLE0604",  # invalid-all-object
+    "PLE0704",  # misplaced-bare-raise
+    "PLE1205",  # logging-too-many-args
+    "PLE1206",  # logging-too-few-args
+    "PLE1310",  # bad-str-strip-call
+    "PLR0904",  # too-many-public-methods
+    "PLR0911",  # too-many-return-statements
+    "PLR0912",  # too-many-branches
+    "PLR0913",  # too-many-arguments
+    "PLR0915",  # too-many-statements
+    "PLR0916",  # too-many-boolean-expressions
+    "PLW0120",  # useless-else-on-loop
+    "PLW0406",  # import-self
+    "PLW0602",  # global-variable-not-assigned
+    "PLW0603",  # global-statement
+    "PLW0711",  # binary-op-exception
+    "PLW1514",  # unspecified-encoding
+    "RET502",  # inconsistent-return-statements
+    "RET505",  # no-else-return
+    "RET507",  # no-else-continue
+    "RET508",  # no-else-break
+    "S102",  # exec-used
+    "SIM118",  # consider-iterating-dictionary
+    "SIM208",  # unneeded-not
+    "UP004",  # useless-object-inheritance
+    "W292",  # missing-final-newline
+    "W605",  # anomalous-backslash-in-string
+]
+
+ignore = [
+    "ARG001",  # unused-argument
+    "F841",  # unused-variable
+    "FIX004",  # fixme
+    "G",  # logging-format-interpolation
+    "G",  # logging-fstring-interpolation
+    "I001",  # ungrouped-imports
+    "I001",  # wrong-import-order
+    "N815",  # invalid-name
+    "PLR0904",  # too-many-public-methods
+    "PLR0911",  # too-many-return-statements
+    "PLR0912",  # too-many-branches
+    "PLR0913",  # too-many-arguments
+    "PLW0603",  # global-statement
+    "PLW1514",  # unspecified-encoding
+    "RET502",  # inconsistent-return-statements
+    "RET505",  # no-else-return
+    "RET507",  # no-else-continue
+    "RET508",  # no-else-break
+    "UP004",  # useless-object-inheritance
+
+    # enabled in pylint but failing in ruff because of too many inline suppressions
+    "A001",  # redefined-builtin
+    "B006",  # dangerous-default-value
+    "B018",  # expression-not-assigned
+    "B018",  # pointless-statement
+    "B023",  # cell-var-from-loop
+    "E501",  # line-too-long
+    "E712",  # singleton-comparison
+    "E721",  # unidiomatic-typecheck
+    "E722",  # bare-except
+    "F401",  # unused-import
+    "F403",  # wildcard-import
+    "F811",  # function-redefined
+    "F811",  # reimported
+    "F821",  # undefined-variable
+    "N804",  # bad-classmethod-argument
+    "N805",  # no-method-argument
+    "N805",  # no-self-argument
+    "PGH001",  # eval-used
+    "PIE790",  # unnecessary-pass
+    "PLR0915",  # too-many-statements
+    "PLW0602",  # global-variable-not-assigned
+    "SIM118",  # consider-iterating-dictionary
+]
```

Comment:
Suggestion: would it make sense to move these configs to `pyproject.toml -> [tool.ruff]`, to start using a common place for such configurations?